tasks.py

```diff
@@ -199,7 +199,7 @@ def validate_kind_version():
     "protocol": "Pre-configure MetalLB with the specified protocol. "
                 "Unconfigured by default. Supported: 'bgp','layer2'",
     "node_img": "Optional node image to use for the kind cluster (e.g. kindest/node:v1.18.19)."
-                "The node image drives the kubernetes version used in kind."
+                "The node image drives the kubernetes version used in kind.",
```

Comment:
This probably belongs in the previous commit?

Reply:
Yes, my bad. I messed up when trying to split the commits.
Same for the comment below